frontend/src/createStream/StreamingTable.jsx

```diff
@@ -1,12 +1,12 @@
 import React from "react";
 import { Grid } from "@material-ui/core";
 
-import axios from "axios";
-
-import PropTypes from "prop-types";
 import SelectDevicesTable from "./SelectDevicesTable";
 import StreamButton from "../general/Buttons/StreamButton";
 
+import * as deviceApi from "../api/DeviceApi";
+// import { createStream } from "../api/StreamApi";
```

Comment:
create stream function does not exist in Stream Api yet. Was added in pr on auth branch